### PR TITLE
Jd/retry intents

### PR DESCRIPTION
**Pull request by ZemindJan**

### CODE REVIEW

**Review by BellringerQuinn — CHANGES_REQUESTED** [review]

**Review by BellringerQuinn — APPROVED** [review]
Thank you! This will resolve a pesky integration/user issue that comes up now and then

**Review by colezemind — CHANGES_REQUESTED** [review]

**Review by colezemind — APPROVED** [review]
Looks good!

File: Plugins/SequencePlugin/Source/SequencePlugin/Private/RequestHandler.cpp [hunk 1]
```diff
@@ -132,32 +134,51 @@ void URequestHandler::ProcessAndThen(TFunction<void(UTexture2D*)> OnSuccess, FFa
...
-
-			if (bWasSuccessful)
+			OnSuccess(Response->GetContentAsString());
+		}
+		else
+		{
+			if(!Response.IsValid())
 			{
-				OnSuccess(Response->GetContentAsString());
+				OnFailure(FSequenceError(RequestFail, "Request is invalid" + Response->GetContentAsString()));
```
Comment (colezemind) [inline Plugins/SequencePlugin/Source/SequencePlugin/Private/RequestHandler.cpp:157]: This line may be referencing Response when Response isn't valid

**Review by colezemind — COMMENTED** [review]
Just one change request required, looks like a response may have been getting referenced when it was invalid

**Review by colezemind — APPROVED** [review]
Changes look good!

**Pull request merged**